orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/pipeline/JenkinsStage.groovy

```diff
@@ -17,72 +17,10 @@
 package com.netflix.spinnaker.orca.igor.pipeline
 
 import groovy.transform.CompileStatic
-import groovy.util.logging.Slf4j
-import com.netflix.spinnaker.orca.CancellableStage
-import com.netflix.spinnaker.orca.ExecutionStatus
-import com.netflix.spinnaker.orca.batch.RestartableStage
-import com.netflix.spinnaker.orca.igor.tasks.MonitorJenkinsJobTask
-import com.netflix.spinnaker.orca.igor.tasks.MonitorQueuedJenkinsJobTask
-import com.netflix.spinnaker.orca.igor.tasks.StartJenkinsJobTask
-import com.netflix.spinnaker.orca.igor.tasks.StopJenkinsJobTask
-import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder
-import com.netflix.spinnaker.orca.pipeline.TaskNode
-import com.netflix.spinnaker.orca.pipeline.model.Execution
-import com.netflix.spinnaker.orca.pipeline.model.Stage
-import com.netflix.spinnaker.orca.pipeline.model.Task
-import com.netflix.spinnaker.orca.pipeline.persistence.ExecutionRepository
-import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Component
 
-@Slf4j
 @Component
 @CompileStatic
-class JenkinsStage implements StageDefinitionBuilder, RestartableStage, CancellableStage {
-  @Autowired StopJenkinsJobTask stopJenkinsJobTask
+class JenkinsStage extends BuildStage {
```

Comment:
If the travis stage is just the same thing as the jenkins stage, why do this major refactoring and renaming in orca instead of just making TravisStage extend JenkinsStage? Seems like a much less disruptive change